### ISSUE TITLE
Make context clearer when choosing and configuring filters on Explore page

### ISSUE THREAD

**Issue by zarino** [mysociety/local-intelligence-hub#205]
Testing showed that people found some filters difficult to use a first, because they were unfamiliar datasets and the UI presents very little context about them. "What’s a sensible number to put in this box?". "I just want to see above average constituencies." etc

I wonder whether a way to fix this would be to present more context next to / as part of the filter controls. We already store min, max, and average values, but we don’t present them.

An additional improvement would be to add “above average” and “below average” options to the default numeric comparators!

**Comment by zarino** [comment]
Potential UI for this:

![Apr-03-2023 16-28-24](https://user-images.githubusercontent.com/739624/229556550-82e8fc88-7698-42e9-83d8-e0f24736fd2c.gif)

```html
<details class="mt-3">
  <summary class="text-muted fs-8 fw-bold lh-1">About this dataset</summary>
  <p class="text-muted fs-8 mb-0 mt-2">mySociety’s Composite UK IMD Index, allowing comparisons across England, Scotland, Wales, and Northern Ireland. 1 = lowest decile, most deprived; 10 = highest decile, least deprived.</p>
</details>
```

**Comment by zarino** [comment]
Closed by #288
